supremm/pcparchive.py

```diff
@@ -62,14 +65,29 @@ def get_datetime_from_pmResult(result):
     """
     return get_datetime_from_timeval(result.contents.timestamp)
 
-def extract_and_merge_logs(job, conf, resconf):
+def extract_and_merge_logs(job, conf, resconf, opts):
```

Comment:
please can you also update the summarize_jobs.py script to pass in the opts dictionary to this function. No need to update the commandline options in the summarize_jobs script for this pull request. However it would be interesting to test to see if we get any noticeable performance improvement from using the dlopen version in the multiproccessing mode.